contrib/inventory/freeipa.py

```diff
@@ -3,19 +3,41 @@
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 import argparse
+from distutils.version import LooseVersion
 import json
-from ipalib import api, errors
+import os
+import sys
+from ipalib import api, errors, __version__ as IPA_VERSION
 from six import u
 
 
 def initialize():
     '''
     This function initializes the FreeIPA/IPA API. This function requires
     no arguments. A kerberos key must be present in the users keyring in
-    order for this to work.
+    order for this to work. IPA default configuration directory is /etc/ipa,
+    this path could be overridden with IPA_CONFDIR environment variable.
     '''
 
     api.bootstrap(context='cli')
+
+    if not os.path.isdir(api.env.confdir):
+        print("WARNING: IPA configuration directory (%s) is missing. "
+              "Environment variable IPA_CONFDIR could be used to override "
+              "default path." % api.env.confdir)
+
+    if LooseVersion(IPA_VERSION) == LooseVersion('4.6.2'):
```

Comment:
Hi,

ipa 4.6.3 has already been released and would fail this test. Shouldn't you use >= rather than == ?